apps/swirl-docs/src/components/IconGrid/IconGrid.tsx

```diff
@@ -1,5 +1,6 @@
 import { FunctionComponent } from "react";
```

Comment:
**issue:** Probably out of scope, but there is a weird gap in the grid.
<img width="1211" alt="Bildschirm­foto 2022-11-09 um 09 44 50" src="https://user-images.githubusercontent.com/1019414/200783693-47a4941c-6318-4422-8a0b-7000a635908e.png">

Reply:
Was a Side Effect of the FlipPopover Implementation. Should be fixed now. On this Place should be The Icon Preview